### PR TITLE
PMM-6624 Install older pmm2-client via packages 

### PR DESCRIPTION
**Pull request by puneet0191**

### CODE REVIEW

**Review by Nailya — APPROVED** [review]
looks good

File: pmm/aws-staging-start.groovy [hunk 1]
```diff
@@ -407,20 +407,25 @@ pipeline {
...
+                            sudo yum -y install https://repo.percona.com/yum/percona-release-latest.noarch.rpm
                             if [[ \$CLIENT_VERSION = dev-latest ]]; then
-                                    sudo yum -y install https://repo.percona.com/yum/percona-release-latest.noarch.rpm
-                                    sudo percona-release disable all
-                                    sudo percona-release enable original testing
-                                    sudo yum clean all
-                                    sudo yum -y install pmm2-client
-                                    sudo yum -y update
+                                sudo percona-release disable all
+                                sudo percona-release enable original testing
```
Comment (Sudokamikaze) [inline pmm/aws-staging-start.groovy:413]: Is it really needed to enable testing repo?

File: pmm/aws-staging-start.groovy [hunk 2]
```diff
@@ -407,20 +407,25 @@ pipeline {
...
+                                sudo yum clean all
+                                sudo yum -y install pmm2-client
+                                sudo yum -y update
                             elif [[ \$CLIENT_VERSION = pmm2-latest ]]; then
-                                sudo yum -y install https://repo.percona.com/yum/percona-release-latest.noarch.rpm
                                 sudo yum clean all
                                 sudo yum -y install pmm2-client
                                 sudo yum -y update
+                                sudo percona-release disable all
```
Comment (Sudokamikaze) [inline pmm/aws-staging-start.groovy:421]: Shouldn't enable of repos be before installation of package?

File: pmm/aws-staging-start.groovy (hunk 1 above)
Comment (puneet0191) [reply pmm/aws-staging-start.groovy:413]: yes because we want to test upgrade of client

**Review by Sudokamikaze — APPROVED** [review]

**Pull request merged**